docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.5
+LABEL maintainer Jan Koppe <post@jankoppe.de>
+
+ARG PUMPIO__GUID=1337
+ARG PUMPIO__UID=1337
+
+ENV PUMP_LOCATION="/usr/local/pumpio"
+ENV PUMP_GIT_TAG="v4.0.0-beta.5"
+ENV PUMP_GIT_REPO="https://github.com/pump-io/pump.io.git"
+
+RUN apk add --no-cache graphicsmagick openssl nodejs python make g++ git \
+     && mkdir -p "${PUMP_LOCATION}" \
+     && git clone --recursive --depth 1 --branch "${PUMP_GIT_TAG}" "${PUMP_GIT_REPO}" "${PUMP_LOCATION}" \
+     && cd "${PUMP_LOCATION}" \
+     && sed -i 's/"bcrypt": "0.8.x"/"bcrypt": "^1.0.2"/g' package.json\
+     && npm install \
+     && npm run build \
+     && cd node_modules/databank \
+     && npm install databank-mongodb \
+     && addgroup -S -g "${PUMPIO__GUID}" "pumpio" \
+     && adduser -S -D -H -G "pumpio" -h "/usr/local/pumpio" -u "${PUMPIO__UID}" "pumpio" \
+     && chown -R "pumpio:pumpio" /usr/local/pumpio \
```

Comment:
No, I was. Though I think I got it confused - `/usr/local/pumpio` is supposed to be the place where the source is, right? Not where `datadir` is? In that case it should be left as `root:root`. No reason to make it owned by the pump user and it increases attack surface.